nixnet/database/database.py

```diff
@@ -16,7 +16,18 @@
 
 
 class Database(object):
-
+    """Opens a database file.
+
+    When an already open database is opened, this function grants access to the
+    same database and increases an internal reference counter. A multiple
+    referenced (open) database must be closed as many times as it has been
+    opened. Until it is completely closed, the access to this database remains
+    granted, and the database uses computer resources (memory and handles). For
+    more information, refer to :any:`nixnet.database.database.Database.close`.
+
+    Args:
```

Comment:
It may seem strange to put Args here, but I did that because our current Sphinx configuration ignores init docstrings (I created issue #232 ). It looks good on the generated docs though since the class header on readthedocs does show the constructor args.